src/capidle/script/isGuessCorrect.js

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Are you able to rename this file to `is-guess-correct` to keep it consistent with the other files?